evm/src/main/java/org/hyperledger/besu/evm/processor/AbstractMessageProcessor.java

```diff
@@ -70,7 +70,7 @@ public abstract class AbstractMessageProcessor {
   // List of addresses to force delete when they are touched but empty
   // when the state changes in the message are were not meant to be committed.
   private final Collection<? super Address> forceDeleteAccountsWhenEmpty;
-  private final EVM evm;
+  protected final EVM evm;
```

Comment:
What if we save the _evm_ reference in `o.h.b.e.p.ContractCreationProcessor` instead of incrementing the visibility here?